### DISCUSSION TITLE
How to automatically update Tables

### DISCUSSION THREAD

**Discussion by PeachesMLG** [j256/ormlite-core#200] [Q&A] [answered]
Hi, im coding a java app using this, and in a newer update ive added a new variable. is there a way to make the table automatically add the new column in the database?

**Comment by j256** [comment, marked as answer]
Sorry no there isn't.  You can use the [`TableUtils.getCreateTableStatements()` method](https://ormlite.com/javadoc/ormlite-core/com/j256/ormlite/table/TableUtils.html#getCreateTableStatements-com.j256.ormlite.support.ConnectionSource-java.lang.Class-) and find the particular column that was added but that doesn't take into account indexes and the like.
